src/main/java/frc/robot/Controls.java

```diff
@@ -253,6 +371,7 @@ private void configureElevatorBindings() {
                         .withTimeout(0.3))
                 .ignoringDisable(true)
                 .withName("Reset elevator zero"));
+    operatorController.rightBumper().whileTrue(s.elevatorSubsystem.holdCoastMode());
```

Comment:
Note from testing- This binding worked, but it appears that most of the resistance is from friction (not brake mode on the motors), so this doesn't make it much easier to move (and it doesn't naturally fall past the first stage).